src/strange_poker.c

```diff
@@ -171,6 +171,23 @@ int main(int argc, char *argv[])
             printMessage( ui, &ctx, "Flop!" );
 
             // ?????
+
+            //decrease player balance after game
+            for( uint32_t i=0; i < playerCnt; i++)
+            {
+                players[i].balance -= players[i].bid;
+            }
+
+            int winnersCnt = 0;
+            int winnersTable[playerCnt];
+            winner_check(playerCnt, players, ctx.tableCards, &winnersCnt, winnersTable);
+            uint64_t winnerMoney = ctx.moneyOnTable / winnersCnt;
+            
+            //add money for winners to balance
+            for( int i = 0; i < winnersCnt; i++)
+            {
+                players[winnersTable[i]].balance += winnerMoney;
```

Comment:
it would be nice if money would not stop from existing.
when you divide `ctx.moneyOnTable / winnersCnt` and add winnersCnt times there is possibility that some money (max `winnersCnt-1`) would just disappear.

Reply:
So how do we want to handle this? Do we want to split the money unequally?